packages/events/defaultSubscribers/dashboard.js

```diff
@@ -9,8 +9,10 @@ module.exports = {
     "rpc:request": [
       function (event) {
         const { payload } = event;
-        if (payload.method === "eth_sendTransaction") {
-          // TODO: Do we care about ID collisions?
+        if (
+          isDashboardNetwork(this.config) &&
```

Comment:
heh sure you can make this wrapper :)

Reply:
by why not instead just check the negation at the very start and short circuit return early if it's not the dashboard network?

Reply:
happy to do that as well - one sec.